File: src/models/canteens.py
from dataclasses import dataclass


@dataclass(frozen=True)
class Canteen:
    id: int
    name: str
    description: str
    address: str
    avatar: str
    current_menu: str

    @classmethod
    def from_row(cls, row):
        return Canteen(
            id=row[0],
            name=row[1],
            description=row[2],
            address=row[3],
            avatar=row[4],
            current_menu=row[5]
        )

    @property
    def json(self) -> dict:
        return {
            "id": self.id,
            "name": self.name,
            "description": self.description,
            "address": self.address,
            "avatar": self.avatar,
            "current_menu": self.current_menu
        }

File: views/js/canteens.js
async function init() {
    let data = await getRequest("/api/canteens")
    let blocks = "";
    for (let i = 0; i < data.length; i++) {
        let canteen = data[i];
        blocks += `
            <div class="container text-center" style="border: solid 1px; margin-top: 25px">
  <div class="row g-2">
    <div class="col-6">
      <img src="${canteen.avatar}" width="150px" style="margin-top: 10px">
    </div>
    <div class="col-6">
      <div class="p-3">
<h5>${canteen.name}</h5>
<p>${canteen.address}</p>
<p>${canteen.description}</p>
<a href="${canteen.current_menu}"><p>Меню</p></a>
      </div>
    </div>
  </div>
</div>
        `
    }

    let canteens = document.getElementById("canteens")
    canteens.innerHTML = blocks;
}

init();